### ISSUE TITLE
Is the project alive?

### ISSUE THREAD

**Issue by szerwi** [benbeezy/Dollo#4]
Hi,

Is this project sill alive? Is the project completed so I can print it and it will work? Is there any documentation with some instructions and list with parts I need to print (I know it is wrote on Github, but maybe it is wrote somewhere else)?

**Comment by mlaga97** [comment]
I don't think this particular repository is active anymore.

This printer is more of an experiment in the philosophy of RepRap then something that should be built with the intent of being able to print anything other than test prints. I did a lot of refactoring and made a lot of changes in my fork over the course of about 3 months, and I managed to get a pretty cool toy that does occasionally spit out plastic in vaguely recognizable forms, but it is certainly a long ways away from being able to print out its own parts. One of the other forks may have gotten farther than I did.

If you want to build one (which will require a lot of tweaking and OpenSCAD skills to make improvements), I would suggest looking at some of the forks:
* [mlaga97/Dollo](https://github.com/mlaga97/Dollo)
* [spegelius/Dollo](https://github.com/spegelius/Dollo)
* [Virindi-AC/Whitney](https://github.com/Virindi-AC/Whitney)

Alternatively, if you want a more mature and more actively developed project with a similar philosophy, I would highly suggest looking into the Hangprinter project.

**Comment by spegelius** [comment]
Yeah, I think I'm the only one working with the original Dollo. Well it has evolved somewhat so a Dollo MK2 or something...
I get relatively good prints out of my Dollo build and use it quite a lot for prints that don't need to be pristine, but still dimensionally accurate. Granted, I'm using metal rods for Z at the moment. Z-axis is the biggest problem of the original design. Im working on some ideas for making it printable with good quality, though.
Also building another Dollo currently, I'm posting info about that here: http://3dprintboard.com/showthread.php?12856-Dollo-3D-Printable-3D-Printer (check tha last pages for up to date info)

**Comment by szerwi** [comment]
Can I print the printer with STLs from this folder: https://github.com/benbeezy/Dollo/tree/master/NEW_long_ties/STL without modyfing anything, or it's impossible? :D

**Comment by spegelius** [comment]
You can print it and it might even work somewhat. But it will need tinkering...

**Comment by szerwi** [comment]
Will the print quality be good? Or I need upgrades like metal Z rod?

**Comment by spegelius** [comment]
Well it seems that the latest commits that were merged from my branch were still about the large gears for z-axis, which causes bad quality: https://www.dropbox.com/s/oowlddhd8ystile/2016-09-06%2012.25.30.jpg?dl=0. That vase should have straight lines and surfaces... the gears just cause the bed to twist around when they turn.
The metal z parts that I designed aren't merged here.

**Comment by szerwi** [comment]
So I should print gear.stl, not gear_large.stl?

**Comment by spegelius** [comment]
Please don't print anything you don't understand, The status in this repository is very much work in progress. I haven't looked at those parts for more than a year and can only remember that it was a PITA to get working. And there are more than one gear to print to get the whole z-axis contraption to work... I don't even remember if I saved the correct sized gears so you'll be printing stuff pretty much blind.

**Comment by szerwi** [comment]
:/ So I think I won't be able to print Dollo. I don't know how to use OpenSCAD.

**Comment by spegelius** [comment]
Not this version, there's no instructions for all parts and some of the parts are most definitely EOL. If you want to build a working Dollo, follow the build I'm doing and hopefully in the end I've explained everything well enough for others to get a working printer-like thing assembled.

**Comment by szerwi** [comment]
Do you know when you will finish the build? And the Dollo that you are building will be fully printable, or it will have some mods like metal Z rods?

**Comment by spegelius** [comment]
Month or two, depending how much time I can use on it. I'm planning on mostly printable, yes. Metal z rods are an option if things don't work out well enough with the printable z.

**Comment by szerwi** [comment]
Ok, so I'm waiting for your finish and instructions :)

**Comment by szerwi** [comment]
@spegelius Are you sure that you will make instructions how to build Dollo when you finish? :) If yes, I will order electronics from China now and start printing Dollo when you will make instructions :)

**Comment by spegelius** [comment]
Well I've already built 1/3 of the new one and also written it down in the forum posts so I very much intend to finish :)
